### PR TITLE
fix: update typos and add profile

### PR DESCRIPTION
**Pull request by WilliamJPriest**
## Fixes Issue
typos in an alt image desc.
fixed some user typos.
added my account details

## Check List (Check all the applicable boxes) 

- [ X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ X] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ X] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1361"></a>

### CODE REVIEW

File: public/data/AbhijithGanesh.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
       "icon": "github"
     },
     {
-      "name": "Conenct with me on LinkedIn",
+      "name": "Connenct with me on LinkedIn",
```
Comment (kumarsonsoff3) [inline public/data/AbhijithGanesh.json:13]: There is a typo again.
```suggestion
      "name": "Connect with me on LinkedIn",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
⚠ There is a Typo again!!

File: public/data/AbhijithGanesh.json (hunk 1 above)
Comment (WilliamJPriest) [reply public/data/AbhijithGanesh.json:13]: oh thank you

**Review by emmalearnscode — APPROVED** [review]
Thanks so much for your contribution William.

The code looks good. For the future I would suggest splitting up your pull requests based on the issue type so you have one for adding your profile and one for fixing typos. This makes it easier to name pull requests and keep track of what changes happen in different pull requests.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: public/data/WilliamJ.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        "icon": "github"
+      },
+      {
+        "name": "laugh and learn with me on twitter" ,
+        "url": "https://twitter.com/WilliamJPriest1",
+        "icon": "twitter"
+      },
+      {
+        "name": "Job Hunt with me on Linkedin",
+        "url": "https://www.linkedin.com/in/william-j-priest-5aa7341a2/",
```
Comment (kabir0x23) [inline public/data/WilliamJ.json:19]: Hey 👋 William,
The 👨‍💻 code looks good 🌠, 
but your LinkedIn URL is broken 💔.

**Review by kumarsonsoff3 — APPROVED** [review]
Good to Goo! 🚀

**Review by kabir0x23 — CHANGES_REQUESTED** [review]
LinkedIn URL needs to be fixed ✅.

**Comment by WilliamJPriest** [comment]
Awesome, and for sure

**Comment by emmalearnscode** [comment]
@WilliamJPriest Did you want to update your linkedin link and typo before we merge this in? You should just be able to make changes to your branch and push them and they will automatically appear here in the pull request. Let us know if you need any help.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable

File: public/data/WilliamJ.json (hunk 2 above)
Comment (WilliamJPriest) [reply public/data/WilliamJ.json:19]: seems to be working when I click on it

Comment (kabir0x23) [reply public/data/WilliamJ.json:19]: > seems to be working when I click on it

image

maybe you are having private account, or some another issues

-> Go to profile:
- Edit your public profile & url:
  - https://www.linkedin.com/public-profile/settings?trk=d_flagship3_profile_self_view_public_profile
  - create a new user name - (if you want to remove numbres from it)
  
 or you can check setting page and explore

Comment (kabir0x23) [reply public/data/WilliamJ.json:19]: you can check profile visibility 

image

**Review by emmalearnscode — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: public/data/WilliamJ.json (hunk 2 above)
Comment (WilliamJPriest) [reply public/data/WilliamJ.json]: Awesome it worked, thank you

**Comment by WilliamJPriest** [comment]
Thanks for you help and guidance everyone

**Pull request merged**